wgengine/router/router_userspace_bsd.go

```diff
@@ -101,9 +102,42 @@ func inet(p netaddr.IPPrefix) string {
 	return "inet"
 }
 
+// See https://github.com/tailscale/tailscale/issues/1307#issuecomment-786045280
+// Remove all IPv6 entries.
+func (r *userspaceBSDRouter) modifiedConfigForFreeBSDBugWorkaround(cfg *Config) *Config {
```

Comment:
this is still mutating the memory of the caller. that is, the returned value aliases cfg in several places: